Drop-In/src/main/java/com/braintreepayments/api/DropInClient.java

```diff
@@ -486,6 +488,33 @@ public void launchDropInForResult(FragmentActivity activity, int requestCode) {
         });
     }
 
+    /**
+     * Called to launch a {@link DropInActivity}.
+     *
+     * NOTE: This method requires {@link DropInClient} to be instantiated with either an Activity
+     * or with a Fragment.
+     *
+     * @see #DropInClient(Fragment, DropInRequest, String)
+     * @see #DropInClient(Fragment, DropInRequest, ClientTokenProvider)
+     * @see #DropInClient(FragmentActivity, DropInRequest, String)
+     * @see #DropInClient(FragmentActivity, DropInRequest, ClientTokenProvider)
+     */
+    public void launchDropIn() {
+        getAuthorization((authorization, authorizationError) -> {
+            if (authorization != null) {
+                if (observer != null) {
+                    DropInIntentData intentData =
+                            new DropInIntentData(dropInRequest, authorization, braintreeClient.getSessionId());
+                    observer.launch(intentData);
+                }
+            } else if (authorizationError != null) {
+                if (listener != null) {
```

Comment:
Same question here. If it's not possible totally fine, just curious.

Reply:
+1